pkg/proc/proc_test.go

```diff
@@ -1815,7 +1815,12 @@ func TestIssue414(t *testing.T) {
 			pc := currentPC(p, t)
 			f, ln := currentLineNumber(p, t)
 			t.Logf("at %s:%d %#x\n", f, ln, pc)
-			err := p.Step()
+			var err error
+			if f == fixture.Source {
```

Comment:
Let's toss in a quick comment here about exactly why we're doing this, as it's non-obvious just from the code.

Reply:
Done.